Zend/zend_API.c

```diff
@@ -5371,49 +5390,44 @@ static zend_string *try_parse_string(const char *str, size_t len, char quote) {
 	return zend_string_init(str, len, 0);
 }
 
-ZEND_API zend_result zend_get_default_from_internal_arg_info(
-		zval *default_value_zval, zend_internal_arg_info *arg_info)
+ZEND_API zend_result zend_get_default_from_arg_info(
```

Comment:
Nit: I realize this was renamed to match the argument, but this is still used only for internal function arg infos. So maybe the existing name still fits?